docs/sources/features/datasources/loki.md

```diff
@@ -189,8 +189,15 @@ datasources:
     jsonData:
       maxLines: 1000
       derivedFields:
-        - datasourceName: Jaeger
+        # Field with internal link pointing to datasource in grafana
+        - datasourceUid: my_jaeger_uid
           matcherRegex: "traceID=(\\w+)"
+          name: TraceID
+          # url will be interpreted as query for the datasource
+          url: "$${__value.raw}"
```

Comment:
This naming is bit weird but not sure how to make it better. Could have 2 separate field but that would need more logic on frontend cause you want to keep the value if you just switch between internal external link, we just change the input field label but keep the content. Could be also named `urlOrQuery`? Feel free to suggest better naming.

Reply:
Which naming exactly? "url"?

Reply:
@aknuds1 yes that one. In UI it's ok nobody sees it but here it may be a bit confusing that why I added that comment.

Reply:
@bergquist What do you think?

Reply:
I guess we consider this feature kinda early stage/experimental. In that case I would not think so much about it. It's only a jsondata field after all. 

We can always rename this once we start polishing this feature and add it to the UI. (in which case I think the name of the field and panel text should match)